.dockerignore

```diff
@@ -3,6 +3,7 @@ dist/
 docs/
 images/
 output/
+helm-charts/
```

Comment:
maybe the installer too?

Reply:
I don't have an installer sub module in my tree.

Reply:
mea culpa. It has been removed.